Playbooks/playbook-Convert_file_hash_to_corresponding_hashes_.yml

```
@@ -0,0 +1,814 @@
+id: Convert file hash to corresponding hashes
+version: -1
+name: 'Convert file hash to corresponding hashes '
```

why do you ha e a space at the end of the name?
```suggestion
name: Convert file hash to corresponding hashes
```